challenges/2021_cloud_base_height/05_optuna_tuning.ipynb

```diff
@@ -0,0 +1,923 @@
+{
```

Comment:
<p>Really nice to include this description of ray.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>